content/docs/get-started/index.md

```diff
@@ -304,88 +272,63 @@ Starting fastapi server...
 
-While the model server is running, you can use your browser to open the Swagger
-UI (OpenAPI) at [http://localhost:8080/docs](http://localhost:8080/docs) and
-check out OpenAPI spec and query examples.
-
-<admon type="info">
-
-MLEM can also generate client-side code to query/infer the model server. Learn
-more about this in
-[Serving User Guide](/doc/user-guide/serving#making-requests).
+[generate client-side]: /doc/user-guide/serving#making-requests
 
 </admon>
 
-## Deploying models to production
+### Deploying models to production
 
-Now, let's take model serving a step further and use production worthy
-deployment technologies. MLEM lets you easily package and deploy your models
-using a variety of platforms like [Docker](/doc/user-guide/deploying/docker),
-[Heroku](/doc/user-guide/deploying/heroku),
-[Sagemaker](/doc/user-guide/deploying/sagemaker) and
-[Kubernetes](/doc/user-guide/deploying/kubernetes), so you don't have to deal
-with the DevOps and implementation details of deployment yourself. See the full
-list in the [Deploying User Guide](/doc/user-guide/deploying).
+MLEM lets you easily deliver your models for testing, shadowing, or production
+with a variety of platforms like [Docker], [Heroku], [Sagemaker] and
+[Kubernetes] (see also the [Deploying] guide). Now you don't have to deal with
+low-level DevOps yourself. Let's look at a few examples:
 
-Let's take a look at a few examples:
+[docker]: /doc/user-guide/deploying/Docker
+[heroku]: /doc/user-guide/deploying/heroku
+[sagemaker]: /doc/user-guide/deploying/sagemaker
+[kubernetes]: /doc/user-guide/deploying/kubernetes
+[deploying]: /doc/user-guide/deploying
 
 <toggle>
 <tab title="Docker container">
 
-### Running a Dockerized model server
-
-#### Building a Docker container
+### Building and running a Dockerized ML model server
 
-First, we use a simple `mlem build` command to build a container image. With
-this one simple command you build a FastAPI model server and package it into a
-Docker container image:
+Let's first use the `mlem build` command to build a containerized model server.
+With this single command, you build a FastAPI server packaged into a Docker
+container image:
 
 ```cli
-$ mlem build docker docker-builder.mlem \
+$ mlem build Docker Docker-builder.mlem \
     --model models/rf \
     --image.name mlem-model
 ⏳️ Loading model from models/rf.mlem
-🛠 Building MLEM wheel file...
-💼 Adding model files...
-🛠 Generating dockerfile...
-💼 Adding sources...
-💼 Generating requirements file...
-🛠 Building docker image mlem-model:latest...
-✅  Built docker image mlem-model:latest
+...
+✅  Built Docker image mlem-model:latest
 ```
 
-This will create a `mlem-model:latest` Docker image, and also a
-[builder specification](/doc/user-guide/building) metafile called
-`docker-builder.mlem` .
+This creates a `mlem-model:latest` Docker image and a builder metafile called
+`Docker-builder.mlem` describing the container.
```

Comment:
```suggestion
`docker-builder.mlem` describing the container.
```

Reply:
Oops. I capitalized these in bulk. Thanks for fix!